cmd/lifecycle/creator.go

```diff
@@ -75,6 +76,11 @@ func (c *createCmd) Args(nargs int, args []string) error {
 			return err
 		}
 	}
+	if c.ParallelExport {
+		if c.CacheImageRef == "" {
+			cmd.DefaultLogger.Warn("parallel export has been enabled, but it has not taken effect because cache image (-cache-image) has not been specified.")
+		}
+	}
```

Comment:
Could we put this validation in the `platform` package? Somewhere in `ResolveInputs`? Our eventual aim is to move all such validations there. That would also have the nice side effect of printing the warning when `cmd/lifecycle/exporter` is invoked with this configuration.